po/de.po

```diff
@@ -1178,72 +1168,63 @@ msgid ""
 msgstr ""
 "* <kbd>Pfeil-links</kbd>: Zur vorherigen Seite navigieren.\n"
 "* <kbd>Pfeil-rechts</kbd>: Zur nächsten Seite navigieren.\n"
-"* <kbd>Strg + Eingabe</kbd>: Führt das Codebeispiel aus, das den Fokus hat.\n"
+"* <kbd>Strg + Eingabe</kbd>: Führe das Codebeispiel aus, das aktuell den Fokus hat.\n"
 "* <kbd>s</kbd>: Suchleiste aktivieren."
 
 #: src/cargo.md:1
-#, fuzzy
 msgid "# Using Cargo"
-msgstr "# Verwenden von Fracht"
+msgstr "# Cargo verwenden"
 
 #: src/cargo.md:3
-#, fuzzy
 msgid ""
 "When you start reading about Rust, you will soon meet [Cargo](https://doc.rust-lang.org/cargo/), "
 "the standard tool\n"
 "used in the Rust ecosystem to build and run Rust applications. Here we want to\n"
 "give a brief overview of what Cargo is and how it fits into the wider ecosystem\n"
 "and how it fits into this training."
 msgstr ""
-"Wenn Sie anfangen, über Rust zu lesen, werden Sie bald [Cargo](https://doc.rust-lang.org/cargo/), "
-"das Standardwerkzeug, kennenlernen\n"
-"Wird im Rust-Ökosystem verwendet, um Rust-Anwendungen zu erstellen und auszuführen. Hier wollen "
-"wir\n"
-"geben einen kurzen Überblick darüber, was Cargo ist und wie es in das breitere Ökosystem passt\n"
-"und wie es in dieses Training passt."
+"Wenn Du anfängst über Rust zu lesen, wirst du sehr bald [Cargo](https://doc.rust-lang.org/"
```

Comment:
Incosistent use of "Du" vs "du".

Reply:
I recently discovered the "suggestion" button in GitHub (the left-most button in the toolbar):

```suggestion
"Wenn du anfängst über Rust zu lesen, wirst du sehr bald [Cargo](https://doc.rust-lang.org/"
```

You can even commit the suggestions yourself afterwards and thus cut down on review round-trips when you simply want to fixup a small detail.

Reply:
🤯  This is amazing! Thanks for mentioning it!